tests/subsys/event_manager/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  event_manager.core:
+    arch_exclude: xtensa
```

Comment:
what is the reason for not supporting xtensa? Is there a plan to fix it?

Reply:
There is a problem with xtensa linker script. It lacks of: #include <snippets-sections.ld>in  soc/xtensa/sample_controller/linker.ld
And because of it compiler does not see linker script added in cmake.

Reply:
Do you know anyone who could comment how we should procced with this problem?

Reply:
Open an issue, they will assign xtensa maintainer